mission-vision-values.md

```diff
@@ -0,0 +1,100 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
+
+OpenTelemetry's Mission: **to enable effective observability through
+high-quality, turnkey, open-source telemetry.**
+
+## Vision (the world we imagine for OTel end-users)
+
+Excellent observability requires high-quality telemetry, as well as the
+performant, consistent instrumentation that makes it possible.
+
+Telemetry in this context is the firehose of raw observational data streaming
+out of software applications, and while “high-quality telemetry” may be a
+requirement for excellent observability, it’s still unreasonable and
+unrealistic to expect developers of application software to add or maintain the
+necessary instrumentation on their own. That is a massive undertaking, and in
+practice it’s not “just code” – there is also necessary alignment around
+protocols and semantic conventions for tags, attributes, and other metadata to
+consider.
+
+So how do we get high-quality, turnkey telemetry without a massive,
+unsustainable engineering effort? This is where OpenTelemetry comes in. To
+achieve our vision, OpenTelemetry sees five key opportunities, listed here:
+
+**Opportunity I: Telemetry should be easy.** With OpenTelemetry we want
```

Comment:
I would drop the word Opportunity from these bullets.

Reply:
Fair point – I think this is what we did for the "Values" below, too.